ideas/121-swarm-compensation.md

```diff
@@ -0,0 +1,72 @@
+## Preamble
+
+    Idea: 121-swarm-compensation
+    Title: Swarm Compensation
+    Status: In Progress
+    Created: 2018-04-02
+    Related: `120-swarm-process`
+
+## Summary
+Ultimately we would like Status to become a public good, owned, developed and ran by it's very own users. In order to achieve that we need to open up our processes for contributions from "the outside" and establish ways for users to propose & incentivise projects they see as important.
+
+Swarms are our current attempt at enabling this. They are ephemeral teams forming to achieve a specified goal and disbanding once achieved or failure becomes apparent. Their defined lifetime and -cycle encourage constant re-evaluation &. evolution.
+
+Compensating contributors for their work is an important component of this and where `120-swarm-process` focuses on establishing a baseline of processes that are open to everyone this idea will focus on experimentally devising compensation mechanisms. 
+
+## Swarm Participants
+
+- Lead Contributor: @martinklepsch
+- Contributor ('payment'): ??? 
```

Comment:
What do we need here? Any other contributors needed?

Reply:
I think if we have one more person who can handle payouts we should probably be good. If the need arises I will try to recruit additional folks but lets start small? 

Maybe both SNT & USD payouts should be possible so not sure who that person would be. Suggestions?

Reply:
Why would we do USD payments? SNT all the way! :P

Reply:
Generally agree but given that some (most?) "core" contributors are paid in USD/EUR suddenly being paid solely in SNT may not be super attractive reducing willingness to participate in these experiments?

Reply:
As I understand, there still exists possibility to convert SNT into USD/EUR, right? So, that shouldn't be an issue.

Reply:
True. But same thing applies. It introduces friction that makes participation in these experiments less attractive. I'll be most happy if we come up with a solution to use just SNT but I don't want to exclude the possibility of USD payouts from the start.